eclair-core/src/test/scala/fr/acinq/eclair/channel/states/g/NegotiatingStateSpec.scala

```diff
@@ -137,6 +137,18 @@ class NegotiatingStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike
     testFeeConverge(f)
   }
 
+  test("recv ClosingSigned (nothing at stake)", Tag("no_push_msat")) { f =>
+    import f._
+    val aliceCloseFee = alice2bob.expectMsgType[ClosingSigned].feeSatoshis
+    alice2bob.forward(bob)
+    val bobCloseFee = bob2alice.expectMsgType[ClosingSigned].feeSatoshis
+    assert(aliceCloseFee === bobCloseFee)
+    val mutualCloseTx = bob2blockchain.expectMsgType[PublishAsap].tx
+    assert(bob.stateData.asInstanceOf[DATA_CLOSING].mutualClosePublished == List(mutualCloseTx))
+    assert(bob2blockchain.expectMsgType[WatchConfirmed].event === BITCOIN_TX_CONFIRMED(mutualCloseTx))
+    alice ! WatchEventSpent(BITCOIN_FUNDING_SPENT, mutualCloseTx)
```

Comment:
It feels like this test isn't finished...don't you want to test that both parties correctly go to the `CLOSED` state once the tx confirms?

Reply:
Fixed in 772ffd9.